cert/cert.proto

```diff
@@ -19,8 +19,8 @@ message RawNebulaCertificateDetails {
     string Name = 1;
 
     // Ips and Subnets are in big endian 32 bit pairs, 1st the ip, 2nd the mask
-    repeated uint32 Ips = 2;
-    repeated uint32 Subnets = 3;
+    RawNetwork Ip = 2;
+    repeated RawNetwork Subnets = 3;
```

Comment:
For a compatible version, I think we can just deprecate the old versions of these fields and make the new Replacement be optional. Then we could have the code prefer the new fields but fall back to the old ones if missing.

Reply:
I like this approach as well

Reply:
Current goals are to get the innards of ipv6 working, figure backwards/forwards compatibility will take the most time so I am ignoring it for now.